hivelink-core/src/main/java/org/apache/iceberg/hivelink/core/HiveMetadataPreservingTableOperations.java

```diff
@@ -282,6 +291,15 @@ private static String getAvroSchemaLiteral(Table table) {
     return schemaStr;
   }
 
+  private static void updateORCStorageDesc(List<FieldSchema> hiveCols, Table table) {
+    String columnsString = hiveCols.stream().map(FieldSchema::getName).collect(Collectors.joining(","));
+    String typesString = hiveCols.stream().map(FieldSchema::getType).collect(Collectors.joining(","));
+
+    Map<String, String> sdParams = table.getSd().getSerdeInfo().getParameters();
```

Comment:
This just creates an in-memo map the of the sdParams but you didn't persist them to HMS right?

Reply:
No, it's getting a reference to the map of the sdParams, so it will set them correctly. Updated the unit test to confirm that.

I did notice though that it could cause NPE if serdeInfo or parameters are null, so added a null check for those.

Reply:
Got it, your first point makes sense.
But if the serdeInfo or parameters are null, shouldn't we create a new map and set the map as the table's serdeInfo/parameters ?